zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -1295,9 +1292,6 @@ impl<F: Field> ExecutionConfig<F> {
             ExecutionState::ErrorDepth => {
                 assign_exec_step!(self.error_depth)
             }
-            ExecutionState::ErrorContractAddressCollision => {
-                assign_exec_step!(self.error_contract_address_collision)
-            }
```

Comment:
since ErrDepth will be processed inside create gadget and call gadget, we can remove this code too?

```
            ExecutionState::ErrorDepth => {
                assign_exec_step!(self.error_depth)
            }
```

Reply:
fixed in [4147e93](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1425/commits/4147e939a01cbdc1ec0a3632bac7f12091e8f85b)